### PR TITLE
MNT #1247 implement changes

### PR DESCRIPTION
**Pull request by prjemian**
as suggested in PR #1247, these changes missed the merge

### CODE REVIEW

File: bluesky/suspenders.py [hunk 1]
```diff
@@ -533,7 +533,8 @@ class SuspendWhenChanged(SuspenderBase):
     
     Only resume if allowed AND when monitored equals expected.
     
-    .. note: this suspender is designed to require bluesky restart if value changes
+    .. note:
```
Comment (danielballan) [inline bluesky/suspenders.py]: I think this needs a blank line below `.. note` to render correctly in HTML.

**Comment by danielballan** [comment]
Oops, apologies for jumping the gun on that merge. See one comment.

File: bluesky/suspenders.py (hunk 1 above)
Comment (prjemian) [reply bluesky/suspenders.py]: Of course.

File: bluesky/suspenders.py [hunk 2]
```diff
@@ -533,7 +533,9 @@ class SuspendWhenChanged(SuspenderBase):
...
+
+       This suspender is designed to require bluesky restart if value changes
```
Comment (mrakitin) [inline bluesky/suspenders.py]:
```suggestion
       This suspender is designed to require bluesky restart if value changes.
```

Comment (mrakitin) [reply bluesky/suspenders.py]: Left a minor suggestion to have this block as a complete sentence.

Comment (prjemian) [reply bluesky/suspenders.py]: done

**Pull request merged**